src/serac/physics/nonlinear_solid.cpp

```diff
@@ -351,6 +355,8 @@ NonlinearSolid::InputOptions FromInlet<NonlinearSolid::InputOptions>::operator()
   result.mu = base["mu"];
   result.K  = base["K"];
 
+  result.viscosity = base["viscosity"];
```

Comment:
Should we automatically make this zero if it's not defined or does the underlying c++ code do that?

Reply:
I have it defaulted to zero:
https://github.com/LLNL/serac/blob/cd1a05753c091eb0b4d48c9c4875fab3fee9b132/src/serac/physics/nonlinear_solid.cpp#L295